frontend/src/components/StockData/StockData.jsx

```diff
@@ -74,22 +76,28 @@ export default function StockData(props) {
     return (
         <div className="stock-data">
             {isLoading ? (
-                <p>Loading...</p>
+                <div className="loading">
+                    <div className="ball"></div>
```

Comment:
nice! could you include how this looks in your PR description? (for future, adding a video walkthrough of your changes in PR description is helpful for reviewers to see E2E functionality working)

Reply:
I tried doing a ball animation for the loading but it wasn't working. I forgot to delete the div for it but I'll make sure to show a video if I do end up doing it.